### DISCUSSION TITLE
Read only view also for users with write access

### DISCUSSION THREAD

**Discussion by julian-forster** [outline/outline#3998] [Ideas]
Often I just want to access the information in the documents.
But with write permissions, i could accidentally edit files.
It would be great to add the feature (maybe controlled under settings) that every document is read only and those users with write permissions have an edit button to change the document

**Comment by tommoor** [comment]
See #3132

**Reply by julian-forster** [reply]
thank you for the quick response. Sorry for opening another discussion which already exists.

**Reply by tommoor** [reply]
No problem, not always easy to find
